WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormInputViewModel.swift

```diff
@@ -11,14 +19,54 @@ final class ShippingLabelCustomsFormInputViewModel: ObservableObject {
     ///
     @Published var contentsType: ShippingLabelCustomsForm.ContentsType
 
-    /// Input customs forms to be updated
+    /// Description of contents, required if contentsType is other.
+    ///
+    @Published var contentExplanation: String
+
+    /// Restriction type of items to be declared in the customs form.
+    ///
+    @Published var restrictionType: ShippingLabelCustomsForm.RestrictionType
+
+    /// Description of restriction type, required if the type is other.
+    ///
+    @Published var restrictionComments: String
+
+    /// International tax number for package.
```

Comment:
Same as above, just need to update the comment to refer to "internal transaction number":

```suggestion
    /// Internal transaction number for package.
```

Reply:
This has been fixed in f61a9ab37e20171c4fb9ebf561f5fb58497725a6.